vss_invoice_mgmt/src/pages/business/pages/masters/vendormaster/pages/VendorMasterList.jsx

```diff
@@ -0,0 +1,202 @@
+  };
+
+  const handleBlock = (id) => {
+    navigate(`/masters/vendormaster/${id}`);
+  };
+
+  const handleSearch = () => {
+    setCurrentPage(currentPage);
+    getVendorList(1, searchName, searchStatus);
+  };
+
+  return (
+    <>
+      <Sidebar />
+      <div className="relative md:ml-64 bg-blueGray-100">
+        <AdminNavbar />
+        <HeaderStats />
+        <div className="px-4 md:px-10 mx-auto w-full -m-24">
+          <div className="relative flex flex-col min-w-0 break-words w-full mb-6 shadow-lg rounded-lg bg-blueGray-100 border-0">
+            <div className="rounded-t bg-white mb-0 px-6 py-6">
+              <div className="text-center flex justify-between">
+                <h6 className="text-blueGray-700 text-xl font-bold">
+                  Vendor Master List
+                </h6>
+
+                <NavLink
+                  className="bg-lightBlue-500 text-white active:bg-lightBlue-600 font-bold uppercase text-xs px-4 py-2 rounded shadow hover:shadow-md outline-none focus:outline-none mr-1 ease-linear transition-all duration-150"
+                  to="/masters/vendormaster"
+                >
+                  Add Vendor Master
+                </NavLink>
+              </div>
+              <br />
+              <div className="container">
+                <div className="grid grid-cols-1 lg:grid-cols-3 gap-6 mt-1 mb-4">
+                  <TextField
+                    id="search-1"
+                    label="Search Vendor Name"
+                    type="search"
+                    onChange={(e) => setSearchName(e.target.value)}
+                  />
+                  <StatusSelect
+                    id="search-2"
+                    label="Search Vendor Status"
+                    onChange={(e) => setSearchStatus(e.target.value)}
+                    value={searchStatus || ""}
+                  />
+                  <Button
+                    variant="contained"
+                    className="button"
+                    onClick={handleSearch}
+                  >
+                    Search
+                  </Button>
+                </div>
+              </div>
+            </div>
+            <div className="container mx-auto px-4">
+              {loading ? (
+                <div className="flex justify-center mt-4">
+                  <CircularProgres />
+                </div>
+              ) : Array.isArray(vendormasterlist) &&
+                vendormasterlist.length > 0 ? (
+                <table className="table-auto w-full">
+                  <thead className="bg-gray-200">
+                    <tr>
+                      {tableHead.col.map((item, index) => (
+                        <th key={index} className="border px-4 py-2 text-left">
+                          {item}
+                        </th>
+                      ))}
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {vendormasterlist.map((item, index) => (
```

Comment:
Use proper variable name instead of item.

Reply:
changed